source/chapters/user_interface.rst

```diff
@@ -440,6 +440,18 @@ Mixxx and is **hidden** in the default
 :ref:`Mixxx user interface <interface-overview>`. Click the gear icon in the upper right corner, then toggle :menuselection:`Skin Settings --> Vinyl Control Options`, or use the specific :ref:`appendix-shortcuts` to show or hide the
 section.
 
+**Vinyl button**
+
+This button enables :term:`vinyl control`. When enabled in Mixxx, the track
+on the Mixxx deck is controlled by the external vinyl control.
+
+**Pass button**
+
+This button enables passthrough. When enabled, audio from an external vinyl
+can be played on the Mixxx deck. To be able to enable this button, you have
+to select an input device for the passthrough control in
+:menuselection:`Preferences --> Sound Hardware`.
```

Comment:
Isn't that true for all buttons in the section? Maybe we should move that information to to parent section or remove it completely (because it already contains a reference to the vinyl control documentation).

Reply:
That is ncessary for VINYL and PASSTHROUGH only.
True, we should put move that note here and instead move the 'See also' block to the top of this section and link to https://manual.mixxx.org/2.2/en/chapters/vinyl_control.html#vinyl-control

Reply:
Yes, the note is the vinyl and paasthrough buttons but I only mentioned it for the passthrough button, I'll correct it to include the both of them. @ronso0 don't quite get where you want the "See also" block moved to.

Reply:
`To be able to enable this button, you have
to select an input device for the passthrough control`
I agree with @Holzhaus, that info would be redundant here:
> Maybe we should move that information to to parent section or remove it completely (because it already contains a reference to the vinyl control documentation


__
> don't quite get where you want the "See also" block moved to

Right above **Vinyl button** IMO:
![image](https://user-images.githubusercontent.com/5934199/99007344-504a9100-2544-11eb-8b3b-5f1655cde6ac.png)

Reply:
Done, thank you.